src/components/month/AgendaEventTile.vue

```diff
@@ -3,6 +3,18 @@
     :id="elementId"
     class="agenda__event is-event"
```

Comment:
Currently, the background color is bound to this class. Probably it would be more consistent with other slots, and more power to the implementer, if we remove these two lines:

```
background-color: v-bind(eventBackgroundColor);
color: v-bind(eventColor);
```

and only use those style declarations for the non-custom event? What's your opinion?

Reply:
Thanks, yes of course you are right that's much better. So in this case, I'll create a base agenda__event css class without those two elements (that can be used for both the custom agenda event and the regular agenda event), then the non custom agenda event will have an extra class with just those two elements. Something like this:

.agenda__event {
  display: flex;
  flex-flow: column;
  justify-content: flex-start;
  border-radius: 4px;
  font-size: var(--qalendar-font-2xs);
  margin-bottom: 4px;
  padding: var(--qalendar-spacing);
  cursor: pointer;
  user-select: none;
}

.agenda__event.non-custom-event {
  background-color: v-bind(eventBackgroundColor);
  color: v-bind(eventColor);
}

Is this okay for you?

Reply:
Sounds great!